### PR TITLE
[Discarded] Setup proxy to call staging server with client & server side. Updates playground with documentation

### LINKED ISSUE #49: Setup proxy to call hosted API endpoints on morris from local development
Setup proxy to call hosted API endpoints on morris from local development

### PR DESCRIPTION
**Pull request by nitishdhar**
- Adds proxy setup (for CORS)  to call morris endpoint from client side rendering code with fetch
- Adds examples of calling morris endpoint from server side without needed proxy
- Adds documentation and updates the playground page to show how to use the mock-api and the proxy in coordination
- Updated server stage render to switch to Badge
- Adds a button icon on top right to link to the playground page. Only visible in local development
- Configures jest to mock fetch calls in unit tests
- Adds unit tests for playground handling server side response rendering and client side fetch

**Full Playground View**
EDPN-Playground

**Error State Playground**
Screen Shot 2023-07-29 at 4 10 25 AM

### CODE REVIEW

**Review by jbritain — APPROVED** [review]

**Review by llamaair — APPROVED** [review]

**Comment by nitishdhar** [comment]
Do not merge yet. I need to run some checks on tests being flaky

File: app/playground/page.tsx
Comment (aslink87) [inline app/playground/page.tsx]:
```suggestion
  icons: 'EDPN_logo_dark_background.png',
```
I think I may have told you to use logo_black which is wrong. logo_black isn't used yet. We may never use it, or we may like it for light-theme use in the future.

**Review by aslink87 — APPROVED** [review]
I like this, thanks for making it straightforward.

If we're to use these examples in the future should we maintain a validation interval of 1 second?

**Comment by nitishdhar** [comment]
> I like this, thanks for making it straightforward.
> 
> If we're to use these examples in the future should we maintain a validation interval of 1 second?

I am switching that to not use cache store, in an actual use case, we would leave that to normal NextJs behavior of using cache and make any changes to that behavior as needed depending on the API we are calling and desired experience on the app. It won't be one size fits all - https://nextjs.org/docs/app/building-your-application/data-fetching/fetching-caching-and-revalidating

**Comment by nitishdhar** [comment]
Merged with another PR https://github.com/ed-pilots-network/frontend/pull/65

**Pull request closed without merging**
